src/craft/app/services/SLPluginsService.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Craft;
+
+class SLPluginsService extends PluginsService {
```

Comment:
Is this monkey patching original Craft code? If so, can we have a link to the original?